apitools/scripts/oauth2l.py

```diff
@@ -194,131 +188,152 @@ def FetchCredentials(scopes, client_info=None, credentials_filename=None):
     return credentials
 
 
-class _Email(apitools_cli.NewCmd):
-
-    """Get user email."""
-
-    usage = 'email <access_token>'
-
-    def RunWithArgs(self, access_token):
-        """Print the email address for this token, if possible."""
-        userinfo = apitools_base.GetUserinfo(
-            oauth2client.client.AccessTokenCredentials(access_token,
-                                                       'oauth2l/1.0'))
-        user_email = userinfo.get('email')
-        if user_email:
-            print user_email
-
-
-class _Fetch(apitools_cli.NewCmd):
-
-    """Fetch credentials."""
-
-    usage = 'fetch <scope> [<scope> ...]'
-
-    def __init__(self, name, flag_values):
-        super(_Fetch, self).__init__(name, flag_values)
-        flags.DEFINE_enum(
-            'credentials_format', 'pretty', sorted(_FORMATS),
-            'Output format for token.',
-            short_name='f', flag_values=flag_values)
-
-    def RunWithArgs(self, *scopes):
-        """Fetch a valid access token and display it."""
-        credentials = FetchCredentials(scopes)
-        print _Format(FLAGS.credentials_format.lower(), credentials)
-
-
-class _Header(apitools_cli.NewCmd):
-
-    """Print credentials for a header."""
-
-    usage = 'header <scope> [<scope> ...]'
-
-    def RunWithArgs(self, *scopes):
-        """Fetch a valid access token and display it formatted for a header."""
-        print _Format('header', FetchCredentials(scopes))
+def Email(args):
```

Comment:
made them all private -- they're just paired with the names of the corresponding subcommands.